cli/test/README.md

```diff
@@ -86,34 +86,50 @@ This document includes CLI commands and the expected CLI output in order to help
-|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease arg1`|**UNSUPPORTED** (currently `Fast DDS QoS Profiles Manager exception caught: Unsupported`)|N/A|
-|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease arg1 arg2`|**UNSUPPORTED** (currently `Fast DDS QoS Profiles Manager exception caught: Unsupported`)|N/A|
+|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease infinite`|N/A|N/A|
+|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease 1`|N/A|N/A|
+|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease other`|`ERROR: value 'other' does not match any member types of the union`|N/A|
 |`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease arg1 arg2 help`|N/A|`PARTICIPANT_BUILTIN_DISCOVERY_CONFIG_DURATION_USAGE`|
+|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease 1 500`|**PENDING** (currently `invalid document structure`) [^3]|N/A|
+|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease infinite 500`|**PENDING** (currently `invalid document structure`) [^3]|N/A|
+|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease other 500`|**PENDING** (currently `invalid document structure`) [^3]|N/A|
 |`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.sec`|`ERROR: set command for Participant builtin discovery config <duration>: 'lease' duration type <sec> attribute expects 1 arguments and received 0`|`PARTICIPANT_BUILTIN_DISCOVERY_CONFIG_DURATION_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.sec[]`|`ERROR: Participant builtin discovery config <duration>: 'lease' duration type <sec> attribute must not be keyed []`|`PARTICIPANT_BUILTIN_DISCOVERY_CONFIG_DURATION_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.sec.other`|`ERROR: Participant builtin discovery config <duration>: 'lease' duration type <sec> attribute must be FINAL element`|`PARTICIPANT_BUILTIN_DISCOVERY_CONFIG_DURATION_USAGE`|
-|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.sec arg`|**UNSUPPORTED** (currently `Fast DDS QoS Profiles Manager exception caught: Unsupported`)|N/A|
+|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.sec 10`|N/A|N/A|
+|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.sec infinite`|N/A|N/A|
```

Comment:
`infinite` is only supported in the following case as per the current CLI mockup:

```
fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease infinite
```

Consequently this case is returning:

```suggestion
|`fastddsqosprof file.xml set participant[profile].builtin.discovery_config.duration.lease.sec infinite`|`Fast DDS QoS Profiles Manager exception caught: value 'infinite' does not match any member types of the union`|N/A|
```

Do you consider that infinite should be supported in every case?

Reply:
I would consider useful to include it as long as it is already supported by the library in those cases. Nevertheless, I think that taking into account its functionality, not allowing `infinite` input in `.sec` and `.nanosec` cases is a reasonable restriction and make sense at all.

Reply:
I agree. One thing is what the XSD currently supports and another what it makes sense to support from now on. In this case, I suggest supporting exclusively the `DURATION_INFINITY` as the other definitions are exactly the same.